### DISCUSSION TITLE
[Bug] releases page display error

### DISCUSSION THREAD

**Discussion by tyouzu1** [community/community#18124] [Other Feature Feedback, Questions, & Ideas]
【Has logged】

<img width="1650" alt="image" src="https://user-images.githubusercontent.com/26712711/172783004-8834fbe4-cfbc-478a-9f40-7f9cded3dd3e.png">

<img width="408" alt="image" src="https://user-images.githubusercontent.com/26712711/172782780-d3db7678-e415-47c5-af82-5c38fa45e4b6.png">

<img width="526" alt="image" src="https://user-images.githubusercontent.com/26712711/172782749-d13fb3cb-9184-44a6-b4ee-341556899adf.png">

**Comment by wdzeng** [comment]
Same issue on Microsoft Edge 102.0.1245.33.

![image](https://user-images.githubusercontent.com/39057640/172793223-96b64ab7-08f9-4ad7-990d-9f2e443e71da.png)
